features/support/world/language.rb

```diff
@@ -1,8 +1,13 @@
 module World
   module Language
-    def language_to_locale(language)
-      { 'English' => :en, 'Welsh' => :cy }[language]
+    def language
+      @language
+    end
+
+    def set_language(code)
```

Comment:
Thanks for the detailed explanation @norm! I understand the reason for the usage of the word `code` now.